core/credentials.yml

```diff
@@ -1,6 +1,6 @@
 rest:
```

Comment:
not wrapping `rest` channel for now. This means that we do not have `sender_id` attributes in the logs for integration tests.

Reply:
Actually, since it is really easy, i'll do it now.

Reply:
(Done)